mrp_partial_production/tests/test_mrp_partial_production.py

```diff
@@ -65,38 +66,36 @@ def test_01_produce_partially(self):
         self.assertFalse(moves,
                          'The move has an assigned line')
 
-        # Loading Lines
-        values = wz_brw.on_change_qty(wz_brw.product_qty, [])
-        values = values.get('value')
-        wz_brw.write(values)
+        # Process production
         wz_brw.do_produce()
+        mrp_brw.post_inventory()
 
-        # Check the number available to produce
-        self.assertEqual(mrp_brw.qty_available_to_produce, 0,
-                         'The quantity available to produce must be 0')
-
+        picking_brw.action_assign()
         moves = picking_brw.move_lines.\
-            filtered(lambda a: a.state == 'assigned')
+            filtered(lambda a: a.state == 'partially_available')
 
         # Checking assigned lines
         self.assertTrue(moves,
                         'The move does not have an assigned line')
-
         # Checking the quantity assigned
-        self.assertEqual(moves.product_uom_qty, 4,
+        self.assertEqual(moves.reserved_availability, 4,
                          'The quantity assigned must be 4')
 
         # Validating Picking
-        picking_brw.do_prepare_partial()
-
-        picking_brw.do_transfer()
+        ctx = {
```

Comment:
1. I think you do not need  active_id and active_ids, you can use only one
2. you can use `"active_ids": picking_brw.ids`

Reply:
ready change